include/execution.hpp

```diff
@@ -34,6 +34,18 @@
 #include <stop_token.hpp>
 
 namespace std::execution {
+  inline namespace __connect_awaitable_ {
+    namespace __impl {
+      template <class R_>
```

Comment:
Yeah, I should rename all the template parameters with trailing `_` to have trailing `Id` instead, since these are types that have been encoded to hide their associated namespaces.